<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(140, 198, 63);</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menustock">
    <property name="title">
     <string>Producto</string>
    </property>
    <addaction name="actionProductoNuevo"/>
    <addaction name="actionProductoBuscar"/>
   </widget>
   <widget class="QMenu" name="menuInventario">
    <property name="title">
     <string>Inventario</string>
    </property>
    <addaction name="actionEntrada"/>
    <addaction name="actionSalida"/>
    <addaction name="actionMovimientos"/>
    <addaction name="actionStock_Actual"/>
   </widget>
   <addaction name="menustock"/>
   <addaction name="menuInventario"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionProductoNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionStock">
   <property name="text">
    <string>Stock</string>
   </property>
  </action>
  <action name="actionEntrada">
   <property name="text">
    <string>Entrada</string>
   </property>
  </action>
  <action name="actionSalida">
   <property name="text">
    <string>Salida</string>
   </property>
  </action>
  <action name="actionProductoBuscar">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionMovimientos">
   <property name="text">
    <string>Movimientos</string>
   </property>
  </action>
  <action name="actionStock_Actual">
   <property name="text">
    <string>Stock Actual</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
